main/src/ca/uwaterloo/flix/api/lsp/provider/CodeActionProvider.scala

```diff
@@ -44,11 +44,11 @@ object CodeActionProvider {
     case ResolutionError.UndefinedStruct(qn, ap, loc) if overlaps(range, loc) =>
       mkNewStruct(qn.ident.name, uri, ap)
 
-    case ResolutionError.UndefinedJvmClass(name, ap, _, loc) if overlaps(range, loc) =>
-      mkImportJava(name, uri, ap)
+    case ResolutionError.UndefinedJvmClass(qn, ap, _, loc) if overlaps(range, loc) =>
+      mkImportJava(qn, Nil, uri, ap)
```

Comment:
I think we can change mkImportJava to take a Name.QName as argument, yes?

Reply:
Then we'll need to construct `Name.QName` for `UndefinedJvmClass`, where we only have `name: String`.

Which do you think is better?

Reply:
OK, let's use qn

Reply:
Do we have to construct a fake QName? If so, it may be better to leave as is.